client/src/__tests__/CreateRoseBudThorn.test.jsx

```diff
@@ -0,0 +1,29 @@
+import { render, screen, mockRouter } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import CreateRoseBudThorn from '../components/CreateRoseBudThorn';
+
+test('CreateRoseBudThorn component renders buttons', () => {
+
+    render( 
+    <Router>
+        <CreateRoseBudThorn />
```

Comment:
This is more of a general comment. Do you need to pass down props to this component to get it to work? Think about it!